system/doc/docs.exs

```diff
@@ -6,6 +6,7 @@
     "installation_guide/INSTALL.md": [source: "../../HOWTO/INSTALL.md"],
     "installation_guide/INSTALL-CROSS.md": [source: "../../HOWTO/INSTALL-CROSS.md"],
     "installation_guide/INSTALL-WIN32.md": [source: "../../HOWTO/INSTALL-WIN32.md"],
+    "installation_guide/INSTALL-WIN32-ARM64.md": [source: "../../HOWTO/INSTALL-WIN32-ARM64.md"],
```

Comment:
This line was causing issue when testing documentation in CI (although the file should be there?), should I remove this line for now?